webviz_subsurface/plugins/_rft_plotter/_views/_map_view/_view.py

```diff
@@ -0,0 +1,212 @@
+from typing import Any, Dict, List, Union
+
+import webviz_core_components as wcc
+from dash import Input, Output, State, callback
+from dash.exceptions import PreventUpdate
+from webviz_config.utils import StrEnum, callback_typecheck
+from webviz_config.webviz_plugin_subclasses import ViewABC
+
+from ..._shared_view_element import GeneralViewElement
+from ..._types import ColorAndSizeByType, DepthType, LineType
+from ..._utils import FormationFigure, RftPlotterDataModel
+from ._settings import FormationPlotSettings, MapSettings
+from ._utils import MapFigure
+
+
+class MapView(ViewABC):
+    class Ids(StrEnum):
+        MAP_SETTINGS = "map-settings"
+        FORMATION_PLOT_SETTINGS = "formation-plot-settings"
+        MAP_VIEW_ELEMENT = "map-view-element"
+        FORMATION_PLOT_VIEW_ELEMENT = "formation-plot-view-element"
+        MAP_FIGURE = "map-figure"
+
+    def __init__(self, datamodel: RftPlotterDataModel) -> None:
+        super().__init__("Map")
+        self._datamodel = datamodel
+
+        self.add_settings_group(MapSettings(self._datamodel), self.Ids.MAP_SETTINGS)
+        self.add_settings_group(
+            FormationPlotSettings(self._datamodel), self.Ids.FORMATION_PLOT_SETTINGS
+        )
+
+        map_column = self.add_column()
+        map_column.add_view_element(GeneralViewElement(), self.Ids.MAP_VIEW_ELEMENT)
```

Comment:
I chose not to do this now, partly because I didn't have time before the feature freeze. But also because it's not clear how we should handle the error messages. They are not so nicely formatted if they are set in the the Graph object.